### PR TITLE
Finnish translations for deform

### PR DESCRIPTION
**Pull request by ztane**

### CODE REVIEW

File: deform/locale/fi/LC_MESSAGES/deform.po [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Translations template for deform.
+# Copyright (C) 2013 ORGANIZATION
+# This file is distributed under the same license as the deform project.
+# Antti Haapala <antti.haapala@anttipatterns.com>, 2013.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: deform 2.0a2\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
```
Comment (domenkozar) [inline deform/locale/fi/LC_MESSAGES/deform.po]: Might need to update this field (or delete it).

Comment (ztane) [inline deform/locale/fi/LC_MESSAGES/deform.po]: Hmm, it says `Report-Msgid-Bugs-To`, not `msgstr` :D (hadn't noticed that one even)

**Comment by miohtama** [comment]
Thank you for your pull request.

I am the new Deform project maintainer. I apologize the issue of this pull request not being commented earlier. Deform project is again under activate maintenance.

This pull request is good to go and will be in the next release.

**Comment by ztane** [comment]
@miohtama Report-Msgid-Bugs-To should contain **your** email address in the template, as per   https://www.gnu.org/software/gettext/manual/html_node/Header-Entry.html

**Comment by miohtama** [comment]
Good point! The new maintainer is not very pedantic and could not care less as he is not aware of this header to be used anywhere in the toolchain. Pull requests welcome, though! ;)

**Pull request merged**